requirements.txt

```diff
@@ -1,4 +1,4 @@
-rasa==1.0.9
+rasa==1.4.3
 mailchimp3==3.0.2
 oauth2client==4.1.2
 gspread==3.0.0
```

Comment:
at some point we should look at updating the rest of these as well, and all should be pinned with `==`

Reply:
what do you mean about being pinned with `==`?

Reply:
Sorry i couldn't add a comment on the ones that were `~=` haha. Just that keeping them as `==` will avoid anyone cloning the repo running into issues due to requirements not matching exactly what we had in there, in any case that's for later